.github/workflows/ci.yml

```diff
@@ -1,16 +1,38 @@
 name: CI
 
 on:
+  workflow_dispatch:
   pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
```

Comment:
This will cause PR tests to run automatically once somebody adds the "safe to test" label.